app/controllers/import_controller.rb

```diff
@@ -26,5 +26,35 @@ def import
       end
     end
   end
-end
-  
+
+  def import_portal_activity
```

Comment:
How is this authenticated?  Can any anonymous user import a portal activity?

When LARA communicates with the portal it uses an authentication token. This is how the portal can trust LARA. So I think there needs to be something similar here.  The same token can be used.  LARA knows the token and the portal knows the token too.  The portal would use the Client#app_secret for LARA.  This will require adding a new field to Client: `Client#domain`, so we can figure out which Client is LARA.  So the portal looks at the `:authoring_site_url` configuration variable.  Gets the domain (for example `authoring.concord.org`), and then looks for a matching domain in the list of clients.

Then on the LARA side, LARA would look at all of its Concord::AuthPortal models to see if there is one with a matching `AuthorPortal#secret`

Here is how it works in the other direction:
LARA sends a republish request to the portal:
https://github.com/concord-consortium/lara/blob/staging/app/models/publishable.rb#L87
It adds the Bearer token header for the portal, if you trace it back you can see it is getting this token from `Concord::AuthPortal#secret`

And on the portal side this is checked using the PeerAccess module. It starts here:
https://github.com/concord-consortium/rigse/blob/master/app/controllers/external_activities_controller.rb#L14
PeerAccess looks for a `Client` that has a matching `Client#app_secret`, here:
https://github.com/concord-consortium/rigse/blob/master/app/controllers/peer_access.rb#L17